### PR TITLE
Try out packtracker.io for analyzing Webpack bundle.

### PR DESCRIPTION
**Pull request by jgerigmeyer**
poodles

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,8 +42,10 @@
     "test:py:report-coverage": "$VIRTUAL_ENV/bin/coveralls",
     "test:py:check-coverage": "coverage3 report --fail-under=100",
     "test": "run-s test:js test:py",
+    "packtracker": "packtracker-upload --stats=webpack-stats.json --output-path=dist/prod",
     "build": "webpack --config webpack.dev.js",
-    "prod": "webpack --config webpack.prod.js"
+    "prod": "webpack --config webpack.prod.js",
+    "prod:stats": "webpack --config webpack.prod.js --json > webpack-stats.json"
```
Comment (wlonk) [inline package.json]: Should `webpack-stats.json` be in our `.gitignore`?

Comment (jgerigmeyer) [reply package.json]: Sure -- this command is never run locally (only on CircleCI), so shouldn't be included in any commits anyway, but I think that makes sense.

**Comment by jgerigmeyer** [comment]
> I think this looks good, but can you tell me a bit about what Packtracker is _for_?

A service for analyzing Webpack bundle size on every commit. I'm not 100% sure it's useful, but wanted to try it out. I invited you so you can see it (though right now you have to manually filter by the `packtracker` branch -- it'll be more interesting once it's on `master` and automatically comparing against individual PRs).

File: package.json (hunk 1 above)
Comment (wlonk) [reply package.json]: I'll give a 👍 once that's in, then.

**Pull request merged**